training/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,29 @@
+Cloud Services and Infrastructure in Digital Publishing
+===========================
+
+We use Amazon Web Services in Digital Publishing to host all of our services, as well as our Continuous Integration (CI).
+
+You should be given relevant AWS accounts by your manager or a Tech Lead - speak to your manager if this does not happen soon upon starting with the team. 
+
+
+Introduction to our Infrastructure
+----------------------------
+
+In your first few weeks you should be invited to a number of 'Intro Workshops', one of which will be on the topic of Infrastructure. 
+
+### Pre reading
+
+- Make sure you're familiar with basic AWS terms: 
+    -  [Availability Zones and Regions](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) 
+    - VPCs and Subnets are also good to understand at a basic level
+
+### Materials
+
+The workshop will walk through [these slides (WIP)](https://docs.google.com/presentation/d/1VQjYd6R6xDRluA_fTF4teu97umJam2y--XlewZjeBrU/edit?usp=sharing) over roughly 1 hour, with time for questions.
+
+
```

Comment:
I wonder if it might also be beneficial to have a next steps/recommended next training section. That way we can build more of a journey through the training if there is a logical flow.

I was also thinking we could have more modules in here that are self learning g stuff as well, so potentially the learning about micro services could actually be a module where we give background of our use then provide the link to an overview of micro services and the 12 factor docs for them to read. Our docs here then would just be framing the learning.